src/components/node/stanford-page/home-page/today-hours.tsx

```diff
@@ -91,7 +91,7 @@ const LibrariesTodayHours = ({libraries, ...props}: {libraries: HoursProps["libr
               </div>
               {library?.suLibraryHours && <TodayLibraryHours branchId={library.suLibraryHours} />}
               {library?.map && (
-                <span className="float-right inline w-auto">
+                <span className="relative top-0 float-right inline w-auto xl:top-[-25px]">
```

Comment:
You shouldn't need `top-0`.
Can we change the float and the inline? maybe use flex around this and the other element?

Reply:
The `top-0` is no longer there.